test/extensions/bootstrap/wasm/test_data/BUILD

```diff
@@ -3,15 +3,30 @@ load(
     "envoy_cc_library",
     "envoy_package",
 )
-load("//bazel/wasm:wasm.bzl", "wasm_cc_binary")
+load("//bazel/wasm:wasm.bzl", "wasm_cc_binary", "wasm_rust_binary")
+load("@io_bazel_rules_rust//rust:rust.bzl", "rust_library")
 
 licenses(["notice"])  # Apache 2
 
 envoy_package()
 
-filegroup(
-    name = "modules",
-    srcs = glob(["*.wasm"]),
+rust_library(
+    name = "proxy_wasm",
+    srcs = ["proxy_wasm.rs"],
```

Comment:
I don't have a better place to comment on this, but this `proxy_wasm.rs` is NOT Proxy-Wasm.

You can copy the contents of that file into `logging_rust.rs`, since it implements the ABI directly... or at the very least use a different name to avoid confusion.

Ideally, we should rewrite this test to use `proxy-wasm-rust-sdk`, but that's probably too big change, and should be done in a separate PR.

Reply:
I will add TODO here and fix when proxy-wasm-rust-sdk bazel support is merged.